app/javascript/packages/document-capture/components/documents-and-selfie-step.tsx

```diff
@@ -93,17 +93,17 @@ interface DocumentsStepValue {
 }
 
 type DefaultSideProps = Pick<
-  FormStepComponentProps<DocumentsStepValue>,
+  FormStepComponentProps<DocumentsAndSelfieStepValue>,
   'registerField' | 'onChange' | 'errors' | 'onError'
 >;
 
-function DocumentsStep({
+export function DocumentsAndSelfieStep({
```

Comment:
`export function` is slightly different than the old `export default` because the import will need curly braces now, do we have a style on that?

Reply:
The other steps all use the non-default style. I changed this one to match, and updated the relevant imports.

Reply:
ok that makes sense! just doing a rough count project-wide, `export default` seems more common

```
> git grep "export default" | wc -l   
     145
> git grep "export function" | wc -l
      45
```

Reply:
> The other steps all use the non-default style. I changed this one to match, and updated the relevant imports.

AFAICT the other steps use the default style?

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/documents-step.tsx#L149

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx#L106

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx#L103

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/in-person-prepare-step.tsx#L85

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/in-person-switch-back-step.tsx#L26

https://github.com/18F/identity-idp/blob/cf000823195acb5f1f7e30fbe8b749bc14bbad9a/app/javascript/packages/document-capture/components/review-issues-step.tsx#L161

Reply:
Changed back.